<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Highscorewindow</class>
 <widget class="QWidget" name="Highscorewindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>High Scores</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QLabel" name="highscore">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="reverse">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>470</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="classic">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="header_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="ranking_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>51</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="classic_1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="classic_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>280</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="classic_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>330</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="classic_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>410</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="classic_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>370</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="classic_6">
   <property name="geometry">
    <rect>
     <x>362</x>
     <y>240</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="classic_9">
   <property name="geometry">
    <rect>
     <x>362</x>
     <y>370</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="classic_10">
   <property name="geometry">
    <rect>
     <x>362</x>
     <y>410</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="classic_8">
   <property name="geometry">
    <rect>
     <x>362</x>
     <y>330</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="classic_7">
   <property name="geometry">
    <rect>
     <x>362</x>
     <y>280</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="ranking_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>570</y>
     <width>51</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="reverse_1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>580</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="header_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>520</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="reverse_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>620</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="reverse_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>670</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="reverse_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>710</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="reverse_5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>750</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="reverse_6">
   <property name="geometry">
    <rect>
     <x>362</x>
     <y>580</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="reverse_7">
   <property name="geometry">
    <rect>
     <x>362</x>
     <y>620</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="reverse_8">
   <property name="geometry">
    <rect>
     <x>362</x>
     <y>670</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="reverse_9">
   <property name="geometry">
    <rect>
     <x>362</x>
     <y>710</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="reverse_10">
   <property name="geometry">
    <rect>
     <x>362</x>
     <y>750</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;

</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
